workshops/forms.py

```diff
@@ -32,21 +35,24 @@ class BootstrapHelperWithoutForm(BootstrapHelper):
 class InstructorsForm(forms.Form):
     '''Represent instructor matching form.'''
 
-    wanted = forms.IntegerField(label='Number Wanted',
-                                initial=INSTRUCTOR_SEARCH_LEN,
-                                min_value=1)
-    latitude = forms.FloatField(label='Latitude',
-                                min_value=-90.0,
-                                max_value=90.0,
-                                required=False)
-    longitude = forms.FloatField(label='Longitude',
-                                 min_value=-180.0,
-                                 max_value=180.0,
-                                 required=False)
-    airport = forms.ModelChoiceField(label='airport',
-                                     queryset=Airport.objects.all(),
-                                     to_field_name='iata',
-                                     required=False)
+    wanted = forms.IntegerField(
+        label='Number Wanted',
+        initial=INSTRUCTOR_SEARCH_LEN,
+        min_value=1)
+    latitude = forms.FloatField(
+        label='Latitude',
+        min_value=-90.0,
+        max_value=90.0,
+        required=False)
+    longitude = forms.FloatField(
+        label='Longitude',
+        min_value=-180.0,
+        max_value=180.0,
+        required=False)
```

Comment:
Lines 38-51 should not be "repaired" this way: we loose information about the author of the original code in the `git blame`. Please revert these changes. (Possibly via `git rebase --interactive HEAD~5` and 'edit' in proper commit e60174d9.)